src/mono/System.Private.CoreLib/src/System/Threading/PreAllocatedOverlapped.Browser.Mono.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace System.Threading
 {
-    public sealed class PreAllocatedOverlapped : System.IDisposable
+    public sealed partial class PreAllocatedOverlapped : System.IDisposable
```

Comment:
I don't think this should be necessary, there might have been an issue before with some files getting incorrectly included when `FeaturePortableThreadPool == false`